<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="TYSS-MF-029" name="com.liqvid.scripts.StudentFunctionalTest" tests="7" failures="0" timestamp="4 Aug 2017 06:07:45 GMT" time="312.170" errors="2">
  <testcase name="verifyStudentHomePage" time="61.250" classname="com.liqvid.scripts.StudentFunctionalTest">
    <error type="com.kirwa.nxgreport.exceptions.NXGReporterStepFailedException">
      <![CDATA[[KIRWA Custom Reporter Step Failed Exception]
	at com.liqvid.library.MyNXGTestListner.onTestSuccess(MyNXGTestListner.java:96)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1695)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1675)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- verifyStudentHomePage -->
  <testcase name="verifyErrorMessageInProfile" time="42.052" classname="com.liqvid.scripts.StudentFunctionalTest"/>
  <testcase name="verifyUploadPhotoStudentProfile" time="42.887" classname="com.liqvid.scripts.StudentFunctionalTest"/>
  <testcase name="verifyInvalidEmailidPhoneNumber" time="28.221" classname="com.liqvid.scripts.StudentFunctionalTest"/>
  <testcase name="functionalityofChangePassword" time="37.718" classname="com.liqvid.scripts.StudentFunctionalTest">
    <error type="com.kirwa.nxgreport.exceptions.NXGReporterStepFailedException">
      <![CDATA[[KIRWA Custom Reporter Step Failed Exception]
	at com.liqvid.library.MyNXGTestListner.onTestSuccess(MyNXGTestListner.java:96)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1695)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1675)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- functionalityofChangePassword -->
  <testcase name="verifyChangePasswordStudentProfile" time="70.862" classname="com.liqvid.scripts.StudentFunctionalTest"/>
  <testcase name="verifyInvalidOldPwd" time="29.180" classname="com.liqvid.scripts.StudentFunctionalTest"/>
</testsuite> <!-- com.liqvid.scripts.StudentFunctionalTest -->
